### PR TITLE
wp standarization

### PR DESCRIPTION
**Pull request by joelrd**
WP Standarization for admin\aioseop_module_class.php up to line 2105

### CODE REVIEW

File: admin/aioseop_module_class.php [hunk 1]
```diff
@@ -1504,266 +1504,442 @@ function get_the_image_by_post_thumbnail( $p = null ) {
...
 		function get_the_image_by_scan( $p = null ) {
-
-			if ( $p === null ) {
+			if ( $p === null )	{
 				global $post;
 			} else {
 				$post = $p;
 			}
-
-			/* Search the post's content for the <img /> tag and get its URL. */
```
Comment (michaeltorbert) [inline admin/aioseop_module_class.php]: @joelrd @amostajo  Why are the comments here removed?

File: admin/aioseop_module_class.php [hunk 2]
```diff
@@ -1504,266 +1504,442 @@ function get_the_image_by_post_thumbnail( $p = null ) {
...
-			preg_match_all( '|<img.*?src=[\'"](.*?)[\'"].*?>|i', get_post_field( 'post_content', $post->ID ), $matches );
-
-			/* If there is a match for the image, return its URL. */
```
Comment (michaeltorbert) [inline admin/aioseop_module_class.php]: @joelrd @amostajo and here.

File: admin/aioseop_module_class.php [hunk 3]
```diff
@@ -1504,266 +1504,442 @@ function get_the_image_by_post_thumbnail( $p = null ) {
...
+								'post_excerpt',
+								$id
+							)
+						)
+					);
 					break;
 				}
 			}
-
-			/* Return the image URL. */
```
Comment (michaeltorbert) [inline admin/aioseop_module_class.php]: @joelrd @amostajo Same thing here. If we're going to remove the comments, it should at least be represented in the docblock.

**Comment by joelrd** [comment]
@michaeltorbert @amostajo  I will add them back in the next Pull-Request with one-line comments and proper spacing.

**Pull request closed without merging**